openedx/core/djangoapps/content_libraries/api.py

```diff
@@ -454,7 +459,11 @@ def create_library(
         )
     except IntegrityError:
         raise LibraryAlreadyExists(slug)  # lint-amnesty, pylint: disable=raise-missing-from
-    CONTENT_LIBRARY_CREATED.send(sender=None, library_key=ref.library_key)
+    CONTENT_LIBRARY_CREATED.send_event(
+        content_library=ContentLibraryData(
```

Comment:
@pomegranited  I'm changing the parameters of this event call. Is this ok?

Reply:
Yep -- we can continue this discussion on https://github.com/openedx/edx-platform/pull/32599/files#r1252390495